internal/pkg/manifest/svc.go

```diff
@@ -375,6 +375,12 @@ func UnmarshalService(in []byte) (interface{}, error) {
 			m.BackendServiceConfig.Image.HealthCheck.applyIfNotSet(newDefaultContainerHealthCheck())
 		}
 		return m, nil
+	case ScheduledJobType:
```

Comment:
It's a bit weird to have `svc.go` to include a `ScheduledJobType`...should we rename this file?

Reply:
Planning to do that in a future PR.